geode-junit/src/main/java/org/apache/geode/security/UpdatableUserAuthInitialize.java

```diff
@@ -28,45 +28,49 @@
  * make sure reset is called after each test to clean things up.
  */
 public class UpdatableUserAuthInitialize implements AuthInitialize {
+  private static final String TEST_UPDATABLE_USER = "test.updatable.user";
+  private static final String TEST_UPDATABLE_WAIT_TIME = "test.updatable.waitTime";
   // use static field for ease of testing since there is only one instance of this in each VM
-  private static final AtomicReference<String> user = new AtomicReference<>();
+  private String user = "";
   // this is used to simulate a slow client in milliseconds
-  private static final AtomicReference<Long> waitTime = new AtomicReference<>(0L);
+  private Long waitTime = 0L;
+
+  @Override
+  public void init(LogWriter systemLogger, LogWriter securityLogger)
+      throws AuthenticationFailedException {
+    user = System.getProperty(TEST_UPDATABLE_USER, "");
+    waitTime = Long.getLong(TEST_UPDATABLE_WAIT_TIME, 0L);
+  }
 
   @Override
   public Properties getCredentials(Properties securityProps, DistributedMember server,
       boolean isPeer) throws AuthenticationFailedException {
     Properties credentials = new Properties();
-    credentials.put("security-username", user.get());
-    credentials.put("security-password", user.get());
+    credentials.put("security-username", user);
+    credentials.put("security-password", user);
```

Comment:
While you're modifying this, you should change from `put` (Map) to `setProperty` (Properties). Both work, but `setProperty` is more correct for Properties.